google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1269,7 +1269,7 @@ def create_file file, path = nil, acl: nil, cache_control: nil,
           path ||= file if file.is_a? String
           raise ArgumentError, "must provide path" if path.nil?
 
-          gapi = service.insert_file name, file, path, options
+          gapi = service.insert_file name, file, path, **options
```

Comment:
I'd prefer to pass keyword arguments directly here, since there is no `delete_if` on the options hash:

```rb
          gapi = service.insert_file name, file, path, acl: File::Acl.predefined_rule_for(acl),
                                                       md5: md5,
                                                       cache_control: cache_control,
                                                       content_type: content_type,
                                                       content_disposition: content_disposition,
                                                       crc32c: crc32c,
                                                       content_encoding: content_encoding,
                                                       metadata: metadata,
                                                       content_language: content_language,
                                                       key: encryption_key,
                                                       kms_key: kms_key,
                                                       storage_class: storage_class_for(storage_class),
                                                       temporary_hold: temporary_hold,
                                                       event_based_hold: event_based_hold,
                                                       user_project: user_project
```